src/lib/components/filter-checkbox.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+  import debounce from 'just-debounce';
+
+  import { goto } from '$app/navigation';
+  import { page } from '$app/stores';
+  import { updateQueryParameters } from '$lib/utilities/update-query-parameters';
+
+  export let parameter: string;
+  export let name: string;
+  export let value: boolean;
+
+  let _value: boolean =
+    !!(parameter && $page.url.searchParams.get(parameter)) || value;
+
+  const id = `${parameter || name}-filter`;
```

Comment:
This falsiness check feels not great here. Code is solid but we should be careful going forward with how this component/API works.

Reply:
I don't plan on keeping this code for very long.